packages/cli/test/functional/test_site/testRelativeLink.md

```diff
@@ -0,0 +1 @@
+**Relative Link Test** This is a relative Intra-Site link (see [Tags](testTags.md))
```

Comment:
let's move this to one of the layout files instead, which is the use case you fixed here

Reply:
Updated the test! But I'm not sure if it is the most appropriate. Let me know if I should make more changes!